### PR TITLE
Blog: generator-m-ionic post - first draft

### PR DESCRIPTION
**Pull request by gruppjo**
Hi @SBoudrias,
thanks for your message. I'm opening this PR as you proposed. 

1. The **images** are currently hosted through my own github, let me know if you want me to change that.
1. Apart form giving a **quick intro** into our generator and its purpose, I wrote this blog article mainly to pick up on **discussions regarding recent development-tool trends** that rather produce *framework-clis* than building flexible, open and extensible systems like [fountainjs](http://fountainjs.io/). I'm mostly referring to these discussions [here](https://github.com/angular/angular-cli/issues/49#issuecomment-206065079) (Angular CLI), [here](https://github.com/driftyco/ionic-cli/issues/845) (Ionic CLI) and [here](https://github.com/yeoman/yeoman/issues/1571) (Yeoman Themes 2016)
1. The **backlinks to the [ionic blog](http://blog.ionic.io/)** are not complete yet, as they're getting published this week. Please don't merge before I updated them!

Let me know what you think and thank you for your consideration.

Excitedly, Jon

### CODE REVIEW

File: app/_posts/blog/2016-07-13-generator-m-ionic-js-fatigue-and-transitioning-to-typescript-angular-2-and-ionic-2.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    <img width="175" src="https://raw.githubusercontent.com/mwaylabs/generator-m-ionic/dev/docs/resources/logo.png">
+  </a>
+</p>
+
+> This is a guest post by [Jonathan Grupp](https://twitter.com/JonathanGrupp)
+
+
+[Generator-M-Ionic](https://github.com/mwaylabs/generator-m-ionic) is an open source collection of advanced workflows for building cross-platform HTML5 mobile apps with Ionic. This article gives a quick introduction into the generator's capabilities and refers to more elaborate resources for those willing to work with it.
+
+As we're transitioning to the next generation technology stack with Typescript, Angular 2 and Ionic 2 we're trying to overcome [JS fatigue](https://medium.com/@ericclemmons/javascript-fatigue-48d4011b6fc4#.vh3bssyd7) and JS tooling fatigue and bring the development of the different tools closer together. You can help us by joining the discussion!
```
Comment (sindresorhus) [inline app/_posts/blog/2016-07-13-generator-m-ionic-js-fatigue-and-transitioning-to-typescript-angular-2-and-ionic-2.md]: Add a comma after `Ionic 2`.

File: app/_posts/blog/2016-07-13-generator-m-ionic-js-fatigue-and-transitioning-to-typescript-angular-2-and-ionic-2.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## What's Generator-M-Ionic for?
+> **Advanced workflows for building rock-solid Ionic apps:** develop, prototype, test, build and deliver high quality apps with Yeoman, Gulp, Bower, Angular, Cordova and of course Ionic. All in one sexy generator.
+
+The roots of [Generator-M-Ionic](https://github.com/mwaylabs/generator-m-ionic) reach back more than two years when my team and I ([@gruppjo](https://github.com/gruppjo)) at [M-Way Solutions](http://www.mwaysolutions.com/) decided to move our HTML5 mobile app development stack to AngularJS and Ionic in [early 2014](http://blog.mwaysolutions.com/2015/03/26/generator-m-the-state-of-html5-mobile-app-development-at-m-way/). Since then [Generator-M-Ionic](https://github.com/mwaylabs/generator-m-ionic) has grown into a full-fledged collection of development tools for building large-scale enterprise apps with [Cordova](http://cordova.apache.org/), [AngularJS](https://angularjs.org/) and [Ionic](http://ionicframework.com/).
```
Comment (sindresorhus) [inline app/_posts/blog/2016-07-13-generator-m-ionic-js-fatigue-and-transitioning-to-typescript-angular-2-and-ionic-2.md]: AngularJS => Angular

consistency

File: app/_posts/blog/2016-07-13-generator-m-ionic-js-fatigue-and-transitioning-to-typescript-angular-2-and-ionic-2.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+### Advanced Workflows
+Handling [CORS & Proxying](https://github.com/mwaylabs/generator-m-ionic/tree/master/docs/guides/cors_proxy.md) issues, managing different sets of [App Icons and splash screens](https://github.com/mwaylabs/generator-m-ionic/tree/master/docs/guides/icons_splash_screens.md), tackling different API endpoints and other things with [Environments](https://github.com/mwaylabs/generator-m-ionic/tree/master/docs/guides/environments.md) and much more is part of the [Advanced Workflows Section](https://github.com/mwaylabs/generator-m-ionic#advanced) of our Guides.
+
+### Ecosystems
+Creating things like backends, push services, user management, build infrastructures and others can be hard. Luckily there's ecosystems like the [Ionic Platform](http://ionic.io/), our own enterprise-targeting [Relution](https://www.relution.io/), which your Generator-M-Ionic project integrates into nicely. [Read more](https://github.com/mwaylabs/generator-m-ionic#ecosystems).
+
+## If that wasn't enough
+... check out our [Github repository](https://github.com/mwaylabs/generator-m-ionic/blob/master/docs/guides/programmatically_change_configxml.md) for the full documentation!
```
Comment (sindresorhus) [inline app/_posts/blog/2016-07-13-generator-m-ionic-js-fatigue-and-transitioning-to-typescript-angular-2-and-ionic-2.md]: GitHub

File: app/_posts/blog/2016-07-13-generator-m-ionic-js-fatigue-and-transitioning-to-typescript-angular-2-and-ionic-2.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+Besides that, we recently had the pleasure of publishing a full blog series about Generator-M-Ionic on the [Ionic Blog](http://blog.ionic.io/). Give it a go for a more tutorial-like approach of getting to know the full value of Generator-M-Ionic:
+
+- Advanced workflows for building rock-solid Ionic apps. Part 1: Playground
+- Advanced workflows for building rock-solid Ionic apps. Part 2: Mountain
+- Advanced workflows for building rock-solid Ionic apps. Part 3: Orbit
```
Comment (sindresorhus) [inline app/_posts/blog/2016-07-13-generator-m-ionic-js-fatigue-and-transitioning-to-typescript-angular-2-and-ionic-2.md]: Can you linkify these.

**Comment by gruppjo** [comment]
Hi @sindresorhus, thanks for your notes. Here's an updated version.

**Comment by sindresorhus** [comment]
LGTM. Nice work on the blog post :)

**Comment by gruppjo** [comment]
:) thank you!

**Comment by SBoudrias** [comment]
LGTM to me too! Thanks for sending this out.

Let us know when you're ready to see this merged and published.

**Comment by gruppjo** [comment]
You're very welcome! Glad it's appreciated :)
I will have no internet access until Monday. I'll include the links until the mid of next week and let you know when everything's ready.

**Comment by gruppjo** [comment]
I added the links and made a final wording change. Ready to publish from my point of view :)

**Comment by SBoudrias** [comment]
Awesome, here it goes!

**Pull request merged**